### PR TITLE
Add Versions to bottom left of app

### PR DESCRIPTION
**Pull request by W3stside**
Adds WEB and CONTRACTS versioning to app in bottom left, flush with operator status on bottom right

On hover:
Screenshot from 2020-12-17 15-27-42

Off hover:
Screenshot from 2020-12-17 15-27-47

### CODE REVIEW

File: src/custom/components/Version/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { ExternalLink, TYPE } from 'theme'
+import { GP_SETTLEMENT_CONTRACT_ADDRESS } from 'custom/constants'
+import { getEtherscanLink } from 'utils'
+
+import { version as WEB, dependencies } from 'src/../../package.json'
+
+const VERSIONS = {
+  WEB,
+  CONTRACTS: dependencies['@gnosis.pm/gp-v2-contracts']
+}
```
Comment (Velenir) [inline src/custom/components/Version/index.tsx:13]: Better do
```ts
import { version: contractsVersion } from '@gnosis.pm/gp-v2-contracts/package.json'
```
to get exact version, not range `^0.0.1-alpha.8`

File: src/custom/components/Version/index.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+const versionsMap = new Map(Object.entries(VERSIONS))
+const versionsList = Array.from(versionsMap)
```
Comment (Velenir) [inline src/custom/components/Version/index.tsx:16]: Why, this is the same
```suggestion
const versionsList = Object.entries(VERSIONS)
```

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr84--gpv2swap.review.gnosisdev.com)**: Testing web app

**Review by Velenir — APPROVED** [review]

File: src/custom/components/Version/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/components/Version/index.tsx:13]: good idea

File: src/custom/components/Version/index.tsx [hunk 3]
```diff
@@ -5,11 +5,12 @@ import { ExternalLink, TYPE } from 'theme'
 import { GP_SETTLEMENT_CONTRACT_ADDRESS } from 'custom/constants'
 import { getEtherscanLink } from 'utils'
 
-import { version as WEB, dependencies } from 'src/../../package.json'
+import { version as WEB } from 'src/../../package.json'
```
Comment (Velenir) [inline src/custom/components/Version/index.tsx:8]: Absolute paths + backtracking :clap:

Comment (Velenir) [reply src/custom/components/Version/index.tsx:8]: Can do this, btw, `@src/../package.json'`

Comment (W3stside) [reply src/custom/components/Version/index.tsx]: lol

File: src/custom/components/Version/index.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+)
+
+const Version = () => {
+  const { chainId } = useActiveWeb3React()
+  const swapAddress = chainId ? GP_SETTLEMENT_CONTRACT_ADDRESS[chainId] : null
+
+  return (
+    <VersionsExternalLink
+      isUnclickable={!chainId || !swapAddress}
+      href={chainId && swapAddress ? getEtherscanLink(chainId, swapAddress, 'address') : ''}
```
Comment (anxolin) [inline src/custom/components/Version/index.tsx:69]: Nice!

**Review by anxolin — COMMENTED** [review]
Awesome!

My only nitpick is, that green looks like it doesn't fit there. Maybe @biocom wants to suggest a color more in line with the theme.

I like the hovering effect and the fact that it show the contract in the selected network!

**Pull request merged**